cl/recap/tasks.py

```diff
@@ -2092,17 +2105,72 @@ def fetch_attachment_page(self: Task, fq_pk: int) -> None:
             "attachment data"
         )
         mark_fq_status(fq, msg, PROCESSING_STATUS.FAILED)
-        return
+        self.request.chain = None
+        return []
     except RECAPDocument.DoesNotExist as exc:
         msg = "Could not find docket to associate with attachment metadata"
         if self.request.retries == self.max_retries:
             mark_fq_status(fq, msg, PROCESSING_STATUS.FAILED)
-            return
+            self.request.chain = None
+            return []
         mark_fq_status(fq, msg, PROCESSING_STATUS.QUEUED_FOR_RETRY)
         raise self.retry(exc=exc)
     msg = "Successfully completed fetch and save."
     mark_fq_status(fq, msg, PROCESSING_STATUS.SUCCESSFUL)
 
+    # Logic to replicate the attachment page to sub-dockets matched by RECAPDocument
+    court_id = rd.docket_entry.docket.court_id
+    if is_appellate_court(court_id):
+        # Subdocket replication for appellate courts is currently not supported.
+        self.request.chain = None
+        return []
+
+    sub_docket_main_rds = list(
+        get_main_rds(court_id, rd.pacer_doc_id).exclude(
+            docket_entry__docket__pacer_case_id=rd.docket_entry.docket.pacer_case_id
```

Comment:
Let's pull the pacer_case_id into its own variable so we don't have to do rd.x.y.z.pacer_case_id here and below.

Reply:
You can also do it for court_id, which is used a bunch above. Maybe worth doing also for pacer_doc_id. Maybe all three defined at the beginning of the function?

Reply:
Sure. Done for `court_id`, `pacer_case_id` and `pacer_doc_id`. 

Thanks!